octopoes/octopoes/connector/octopoes.py

```diff
@@ -19,11 +19,16 @@
 )
 from octopoes.models.exception import ObjectNotFoundException
 from octopoes.models.explanation import InheritanceSection
+from octopoes.models.ooi.findings import Finding, RiskLevelSeverity
 from octopoes.models.origin import Origin, OriginParameter
 from octopoes.models.pagination import Paginated
 from octopoes.models.tree import ReferenceTree
 from octopoes.models.types import OOIType
 
+DEFAULT_SEVERITIES = {severity for severity in RiskLevelSeverity}
+DEFAULT_LIMIT = 5000
```

Comment:
Do we still have locations in rocky where we filter the resultset and need this large a default limit? If not, and all filtering has since been moved to octopoes, a more sane (and quicker) default would probably be useful as we are probably not going to show 5000 items in the frontend.